src/python/WMCore/ReqMgr/DataStructs/Request.py

```diff
@@ -20,6 +20,9 @@
 from WMCore.ReqMgr.DataStructs.RequestStatus import REQUEST_START_STATE
 from WMCore.ReqMgr.DataStructs.RequestError import InvalidSpecParameterValue
 
+#TODO: this need to be update from DB instead of hard coding for more flexibility
+RESULT_FILTER_FOR_DAS = ["RequestName", "OutputDatasets", "InputDatasets", "PrepID"]
```

Comment:
I'd add the following three keys: Campaign, GlobalTag and CMSSWVersion. I'd say we should also add the ConfigCacheID (unless DAS is somehow fetching the configuration that produced that dataset already?), but if you prefer, this can wait until people explicitly ask for this information.

BTW, three questions:
a) why InputDatasets (plural)? Is it for TaskChain cases? Or it adds the Pileup sample?
b) giving a second thought, OutputDatasets sounds useless, since the user using DAS will know already which dataset or block or file he wants to look at
c) PrepID, GlobalTag and CMSSWVersion can be different for TaskChain (and StepChain) requests, are you considering it?